### PR TITLE
Add toDecimal support

### PR DESCRIPTION
**Pull request by Dean-Coakley**
Fixes: https://github.com/Masterminds/sprig/issues/109
Closes: https://github.com/helm/helm/issues/4378

* Add support for `toDecimal` which takes an interface, and returns unix octal format
* Add `toDecimal` unit test

Not sure about edge cases.

### CODE REVIEW

File: docs/conversion.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following type conversion functions are provided by Sprig:
 - `int64`: Convert to an `int64`
 - `toString`: Convert to a string
 - `toStrings`: Convert a list, slice, or array to a list of strings.
+- `toDecimal`: Converts to a unix octal.
```
Comment (technosophos) [inline docs/conversion.md]: Can you update this documentation to be a little more specific? Then I'll get it merged.

Comment (Dean-Coakley) [reply docs/conversion.md]: Found an error that the function is actually called decimal when called from templates so I updated the docs accordingly. https://github.com/Masterminds/sprig/pull/128/files#diff-4b8d8686fe71a23ce9bf8e85babe078dR164

Also added an example and added more specific details.
